src/datasource_toolkit/forestadmin/datasource_toolkit/decorators/chart/result_builder.py

```diff
@@ -2,7 +2,7 @@
 from datetime import date, datetime
 from typing import Dict, List, Optional, TypedDict, Union
 
-from dateutil.relativedelta import relativedelta
+import pandas as pd
```

Comment:
we have a direct dependency to panda, but not to dateutil (it is a dependency of panda), that's why I changed it.